conf.d/main

```diff
@@ -18,13 +18,38 @@ CONF=$WEBROOT/.env
 LOC_SHARE=/usr/local/share
 LOC_BIN=/usr/local/bin
 
+#install inoncube loader
+systemctl restart apache2
+wget http://downloads3.ioncube.com/loader_downloads/ioncube_loaders_lin_x86-64.tar.gz
+tar -xvf ioncube_loaders_lin_x86-64.tar.gz
+cp ioncube/ioncube_loader_lin_8.2.so /usr/lib/php/20220829
+
+#Update php related config files for ioncube loader
+sed -i '2 a zend_extension = "/usr/lib/php/20220829/ioncube_loader_lin_8.2.so"' /etc/php/8.2/apache2/php.ini
+sed -i '2 a zend_extension = "/usr/lib/php/20220829/ioncube_loader_lin_8.2.so"' /etc/php/8.2/cli/php.ini
+sed -i '2 a zend_extension = "/usr/lib/php/20220829/ioncube_loader_lin_8.2.so"' /etc/php/8.2/fpm/php.ini
+sed -i 's/upload_max_filesize =.*/upload_max_filesize = 100M/g' /etc/php/8.2/apache2/php.ini
+sed -i 's/post_max_size =.*/post_max_size = 100M/g' /etc/php/8.2/apache2/php.ini
+sed -i 's/max_execution_time =.*/max_execution_time = 360/g' /etc/php/8.2/apache2/php.ini
+sed -i 's/upload_max_filesize =.*/upload_max_filesize = 100M/g' /etc/php/8.2/cli/php.ini
+sed -i 's/post_max_size =.*/post_max_size = 100M/g' /etc/php/8.2/cli/php.ini
+sed -i 's/max_execution_time =.*/max_execution_time = 360/g' /etc/php/8.2/cli/php.ini 
+sed -i 's/upload_max_filesize =.*/upload_max_filesize = 100M/g' /etc/php/8.2/fpm/php.ini
+sed -i 's/post_max_size =.*/post_max_size = 100M/g' /etc/php/8.2/fpm/php.ini
+sed -i 's/max_execution_time =.*/max_execution_time = 360/g' /etc/php/8.2/fpm/php.ini 
```

Comment:
Again not wrong, but you can skip the fpm config. Also, some of these can be set as env vars in the Makefile. See which ones here: https://github.com/turnkeylinux/common/blob/18.x/mk/turnkey/php.mk

Reply:
And then in the Makefile:
```
PHP_POST_MAX_SIZE=100M
PHP_UPLOAD_MAX_FILESIZE=100M
PHP_MAX_EXECUTION_TIME=360
```
(And actually then you can remove all these lines)